packages/config/src/ProfileInfo.ts

```diff
@@ -491,6 +491,15 @@ export class ProfileInfo {
         return this.mLoadedConfig;
     }
 
+    // _______________________________________________________________________
+    /**
+     * Helper function to identify if the existing config is secure or not
+     * @returns true if the teamConfig is storing credentials securely, false otherwise
+     */
+    public isSecured(): boolean {
```

Comment:
I was in the middle of replying to this comment and got sidetracked.
Sorry for the delay. 😞 

This will apply in the context of the entire Config file (and the ~/.zowe/settings/imperative.json` file)

Reply:
So if one set of credentials are in zowe.config.json in plain text and another set of credentials are stored in the SCS, does this function return 'secure' or 'insecure' ?

Reply:
I am not sure what a consumer would expect. I mostly wanted to be able to give the correct answer should a customer ask.

If we get strong feedback that something different is needed, we could attempt some of your suggestions at that time.